checkov/common/bridgecrew/integration_features/features/custom_policies_integration.py

```diff
@@ -72,5 +81,25 @@ def _get_policies_from_platform(self):
         policies = [p for p in policies if p['isCustom']]
         return policies
 
+    def post_runner(self, scan_reports):
+        if self.bc_cloned_checks.keys():
```

Comment:
```suggestion
        if self.bc_cloned_checks:
```
not sure, if it makes any difference in performance, but there is no need to especially check the `keys`